pythran/pythonic/include/numpy/random/rand.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef PYTHONIC_INCLUDE_NUMPY_RANDOM_RAND_HPP
+#define PYTHONIC_INCLUDE_NUMPY_RANDOM_RAND_HPP
+
+#include "pythonic/include/utils/proxy.hpp"
+#include "pythonic/include/types/ndarray.hpp"
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+      template <class... T>
+      types::ndarray<double, sizeof...(T)> rand(T... shape);
+
+      double rand();
+
```

Comment:
this conflicts with one of the commit of my previous PR... I'll prune it from my PR. (you knew it: you reviewed it ;-)